docs/services/slam/cartographer/_index.md

```diff
@@ -54,188 +81,380 @@ Follow the instructions below to set up the `cartographer` module on your robot:
 
-Go to your robot's page on the [Viam app](https://app.viam.com/).
-Navigate to the **Config** tab on your robot's page and select **Raw JSON** mode.
+{{%expand "Click to view an example JSON configuration for creating a new map" %}}
 
-Note: This template includes configuration for both the `rplidar` and `cartographer` modules, since both are required here.
+See [Attributes](#attributes) for details.
 
-```json
+```json {class="line-numbers linkable-line-numbers"}
 {
-  "components": [
+  "modules": [
     {
-      "name": "<your-rplidar-name>",
-      "model": "viam:lidar:rplidar",
-      "type": "camera",
-      "namespace": "rdk",
-      "attributes": {},
-      "depends_on": []
+      "type": "registry",
+      "name": "viam_rplidar",
+      "module_id": "viam:rplidar",
+      "version": "0.1.14"
+    },
+    {
+      "type": "registry",
+      "name": "viam_cartographer",
+      "module_id": "viam:cartographer",
+      "version": "0.3.36"
     }
   ],
   "services": [
     {
-      "name": "<your-cartographer-name>",
-      "type": "slam",
-      "namespace": "rdk",
-      "model": "viam:slam:cartographer",
       "attributes": {
         "config_params": {
-          "mode": "2d"
+          "max_range_meters": "25",
+          "mode": "2d",
+          "min_range_meters": "0.2"
+        },
+        "camera": {
+          "name": "rplidar",
+          "data_frequency_hz": "5"
         },
-        "data_dir": "</path/to/cartographer-directory>",
-        "data_rate_msec": <int>,
-        "map_rate_sec": <int>,
-        "sensors": [
-          "<your-rplidar-name>"
-        ]
+        "enable_mapping": true,
+        "use_cloud_slam": true
+      },
+      "name": "slam",
+      "type": "slam",
+      "namespace": "rdk",
+      "model": "viam:slam:cartographer"
+    },
+    {
+      "name": "Data-Management-Service",
+      "type": "data_manager",
+      "attributes": {
+        "tags": [],
+        "additional_sync_paths": [],
+        "sync_interval_mins": 0.1,
+        "capture_dir": ""
       }
     }
   ],
+  "components": [
+    {
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+      "service_configs": [
+        {
+          "attributes": {
+            "capture_methods": [
+              {
+                "disabled": false,
+                "method": "NextPointCloud",
+                "capture_frequency_hz": 5
+              }
+            ]
+          },
+          "type": "data_manager"
+        }
+      ],
+      "name": "rplidar",
+      "model": "viam:lidar:rplidar",
+      "type": "camera"
+    }
+  ]
+}
+```
+
+{{% /expand%}}
+
+#### Update existing map
+
+This mode is similar to [Create new map](#create-new-map), except it creates a new version of an existing map in the robot's location. (It does not overwrite the old version of the map.)
+
+The configuration is similar to the configuration for [creating a new map](#create-new-map), except you additionally configure the following **Attribute**:
```

Comment:
```suggestion
The configuration is similar to the configuration for [creating a new map](#create-a-new-map), except you additionally configure the following **Attribute**:
```

Reply:
This sentence is also almost identical to the preceding. Might it be possible to combine them?